### DISCUSSION TITLE
Welcome to microsoft_todo Discussions!

### DISCUSSION THREAD

**Discussion by yunushabeeb** [d-code-h/microsoft_todo#25] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by estherkuranga** [comment]
Hi😊 everyone, I'm moontess, currently learning web dev(a junior developer if I may say) and will be fixing the responsiveness of the web page. But to be honest, it's really nice to be in this community. Being here  has been helping and beneficial. 
Thank you Boss🙌 and everyone for the love💘🙇‍♂️🥰
Catch ya in high places🥂✌🤗

**Reply by yunushabeeb** [reply]
Yeah. Thanks @Moontess

Free free to discuss on a share your views on the project here.

**Comment by yunushabeeb** [comment]
Okay. You know what guys.

This discussion forum will only serve announcement purposes.

New discussion forum is on the way for questions and others.

Stay tuned.

**Reply by codelizz** [reply]
Alright boss,. You're too much 🥰.
